swagger-spec/collection_provider/detail.yaml

```diff
@@ -0,0 +1,102 @@
+# /provider/collections/{collection_id}/
+get:
+  summary: Collections Providers Detail
+  description: >-
+
+    This returns a single Collections Provider entity.
+
+    #### Permissions
+
+    This information is public.
+
+    #### Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+    The `data` key contains an array of nodes ids.
+
+    The `links` key contains a dictionary of links that can be used for [pagination](#tag/Pagination).
+  parameters:
+    - in: path
+      type: string
+      required: true
+      name: collection_id
+      description: 'A short id for that collection'
+  tags:
+    - Collection Providers
+  operationId: collection_provider_detail
+  x-response-schema: Collection Provider
+  consumes:
+    - application/json
+  responses:
+    '200':
+      description: 'OK'
+      examples:
+        application/json:
+          data:
+            id: testlab
+            type: collection-providers
+            attributes:
+              name: Test Lab (Stage1)
+              description: |-
+                <p><span>&lt;p style="background-color:;color:</span><span>070539</span><span>;"&gt;</span><span><span>Character Lab was founded in 2013 by a scientist and two educators: Angela Duckworth, author of </span></span><em>Grit: The Power of Passion and Perseverance</em><span><span> and Rosa Lee and Egbert Chang Professor at the University of Pennsylvania; Dave Levin, co-founder of KIPP public charter schools; and Dominic Randolph, Head of School at Riverdale Country School. </span><span>&lt;/p&gt;</span></span></p>
+                <span> </span>
+                <p>Dave and Dominic met Angela on a field trip. The two educators were visiting leading researchers around the world, eager to find scientific insights that might help young people lead lives that were good for others as well as themselves. These conversations revealed a problem: because research involving kids and educators was so much more time-consuming and expensive than research with adults, the pace of scientific discovery was frustratingly slow. What’s more, insights that had been uncovered by scientists remained hidden from educators and parents, buried in academic articles filled with technical jargon. </p>
+              advisory_board: ''
+              example:
```

Comment:
I'm not the most familiar with swagger specs, but will these automagically be set to `null`? Otherwise, `example`, `email_support`, and `facebook_app_id` may need to be set explicitly to `null`